modules/ROOT/partials/configuration/advcode.adoc

```diff
@@ -20,3 +20,33 @@ tinymce.init({
   toolbar: 'code',
 });
 ----
+
+[[advcode_prettify_getcontent]]
+== `+advcode_prettify_getcontent+`
+
+include::partial$misc/admon-requires-7.3v.adoc[]
+
+As part of the {productname} 7.3 release, the {pluginname} plugin includes a new option, `advcode_prettify_getcontent`, which is set to `false` by default.
+
+When this option is set to `true` if will format the HTML code when `editor.getContent` is called.
+
+This is equivalent when retrieving formatted content by calling `+tinymce.activeEditor.getContent({ prettify: true })+`, regardless of the option's status.
```

Comment:
```suggestion
As part of the {productname} 7.3 release, the {pluginname} plugin includes a new option, `advcode_prettify_getcontent`, which is set to `false` by default.

When this option is set to `true` it will format the HTML code when `editor.getContent` is called.

This is equivalent to retrieving formatted content by calling `+tinymce.activeEditor.getContent({ prettify: true })+`, regardless of the option's status.
```